components/navigation.css

```diff
@@ -0,0 +1,105 @@
+.main-header {
+    position: sticky;
+    top: 0;
+    z-index: 999;
+    width: 100vw;
+    background: white;
+    overflow: hidden;
+}
+
+.main-nav,
+.form-search .search-container,
+.nav-search,
+.uu-nav-links,
+.uu-nav-links li {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.main-nav {
+    justify-content: space-around;
+    padding: var(--sp-8);
+    width: 100vw;
+    box-shadow: 0 0 5px 1px var(--gray);
+}
+
+.uu-logo {
+    order: -1;
+    height: 6rem;
+    width: 6rem;
+}
+
+.form-search {
+    height: 3rem;
+}
+
+.nav-search {
+    width: 20rem;
+}
+
+.form-search .search-container {
+    height: 100%;
+    width: 100%;
+    border: 1px solid var(--text-color);
+    border-radius: 15px;
```

Comment:
Try to give in rem if it is above 10px

Reply:
Thanks for the suggestion, Samyak! But this will remain 15px no matter the device, didn't want to change it depending on root font-size, that's why in pixels.